queries/futures/useGetFuturesOpenOrders.ts

```diff
@@ -9,61 +9,91 @@ import { ETH_UNIT } from 'constants/network';
 	const { selectedFuturesAddress } = useRecoilValue(futuresAccountState);
-	const { network } = Connector.useContainer();
+	const { network, synthsMap } = Connector.useContainer();
 	const futuresEndpoint = getFuturesEndpoint(network?.id as NetworkId);
 
 	const marketInfo = useRecoilValue(marketInfoState);
 	const setOpenOrders = useSetRecoilState(openOrdersState);
 
-	return useQuery<any[]>(
+	return useQuery<FuturesOrder[]>(
 		QUERY_KEYS.Futures.OpenOrders(network?.id as NetworkId, selectedFuturesAddress),
 		async () => {
 			if (!selectedFuturesAddress) {
 				setOpenOrders([]);
 				return [];
 			}
 			try {
-				const marketAddress = marketInfo?.market;
+				const marketAsset = marketInfo?.assetHex;
 				const response = await request(
 					futuresEndpoint,
 					gql`
-						query OpenOrders($account: String!, $market: String!) {
-							futuresOrders(where: { account: $account, market: $market, status: Pending }) {
+						query OpenOrders($account: String!, $asset: String!) {
+							futuresOrders(where: { abstractAccount: $account, asset: $asset, status: Pending }) {
 								id
 								account
 								size
 								market
 								asset
 								targetRoundId
+								targetPrice
 								timestamp
 								orderType
 							}
 						}
 					`,
-					{ account: selectedFuturesAddress, market: marketAddress }
+					{ account: selectedFuturesAddress, asset: marketAsset }
 				);
 
-				const openOrders = response
-					? response.futuresOrders.map((o: any) => ({
-							...o,
-							asset: ethersUtils.parseBytes32String(o.asset),
-							targetRoundId: new Wei(o.targetRoundId, 0),
-							size: new Wei(o.size).div(ETH_UNIT),
-					  }))
+				const openOrders: FuturesOrder[] = response
+					? response.futuresOrders.map((o: FuturesOrder) => {
+							const asset: FuturesMarketAsset = ethersUtils.parseBytes32String(
+								o.asset
+							) as FuturesMarketAsset;
+							const size = new Wei(o.size).div(ETH_UNIT);
+							const targetPrice = new Wei(o.targetPrice).div(ETH_UNIT);
+							const targetRoundId = new Wei(o.targetRoundId, 0);
+							return {
+								...o,
+								asset: asset,
+								targetRoundId: targetRoundId,
+								targetPrice: targetPrice.gt(0) ? targetPrice : null,
+								size: size,
+								market: getMarketName(asset),
+								marketKey: MarketKeyByAsset[asset],
+								orderType: o.orderType === 'NextPrice' ? 'Next-Price' : o.orderType,
+								sizeTxt: formatCurrency(asset, size.abs(), {
+									sign: asset ? synthsMap[asset]?.sign : '',
+									minDecimals: size.abs().lt(0.01) ? 4 : 2,
+								}),
+								targetPriceTxt: formatCurrency(asset, targetPrice, {
+									sign: '$',
+									minDecimals: targetPrice.lt(0.01) ? 4 : 2,
+								}),
+								side: size.gt(0) ? PositionSide.LONG : PositionSide.SHORT,
+								isStale:
+									o.orderType === 'NextPrice' &&
+									wei(marketInfo?.currentRoundId ?? 0).gte(wei(o.targetRoundId).add(2)),
```

Comment:
As there's a few of these `wei(marketInfo?.currentRoundId ?? 0)` it might be nice to do a single `const currentRound = wei(marketInfo?.currentRoundId ?? 0)` above